cypress.config.js

```diff
@@ -0,0 +1,33 @@
+const { defineConfig } = require('cypress')
+
+module.exports = defineConfig({
+  chromeWebSecurity: false,
+  env: {
+    HOST: 'https://ffiec.cfpb.gov',
+    ENVIRONMENT: 'NOT CI',
+    USERNAME: 'frontend.testing@mailinator.com',
+    INSTITUTION: 'FRONTENDTESTBANK9999',
+    TEST_DELAY: 1000,
+    ACTION_DELAY: 1000,
+    AUTH_REALM: 'hmda2',
+    AUTH_CLIENT_ID: 'hmda2-api',
+  },
+  defaultCommandTimeout: 10000,
+  projectId: 'uk89dv',
+  retries: {
+    runMode: 1,
+    openMode: 0,
+  },
+  experimentalStudio: true,
+  e2e: {
+    // Excludes file that allows us to run all tests in the UI
+    // Ensures PROD doesn't run all tests twice
+    excludeSpecPattern: !process.env.CYPRESS_ENVIRONMENT
+      ? ['cypress/e2e/all/index.cy.js']
+      : [],
+    setupNodeEvents(on, config) {
+      return require('./cypress/plugins/index.js')(on, config)
+    },
+    specPattern: 'cypress/e2e/**/*.{js,jsx,ts,tsx}',
+  },
```

Comment:
Does the upgraded Cypress version still support the `--spec <list_of_tests>` option? Or does this `specPattern` determine which tests files are executed?
 
We currently use an environment variable with `--spec` to decide which tests to run so I'm just trying to understand if that becomes obsolete or simply updated to reflect the new directory structure.

Reply:
Looks like Cypress v11.2.0 re-added the ability to run all spec files when doing `cypress open` aka via the browser. Going to remove this code and the `e2e/all/` directory as it's no longer needed.